src/components/SearchResults/SearchResults.tsx

```
@@ -0,0 +1,220 @@
+                    probabilityFactor < 1.2
+                  ) {
+                    probabilityClass = "high";
+                    probabilityText = "High Probability";
+                  } else if (probabilityFactor > 1.2) {
+                    probabilityClass = "veryhigh";
+                    probabilityText = "Very High Probability";
+                  }
+
+                  console.log(result);
+                  return (
+                    <tr>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)